alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/AlertingRestTestCase.kt

```diff
@@ -278,7 +278,6 @@ abstract class AlertingRestTestCase : ESRestTestCase() {
     override fun restClientSettings(): Settings {
         return if (isDebuggingTest || isDebuggingRemoteCluster) {
             Settings.builder()
-                    .put(CLIENT_RETRY_TIMEOUT, TimeValue.timeValueMinutes(10))
```

Comment:
Should we increase the connect timeout as well?

Reply:
What connect timeout do you mean? Do you mean `CLIENT_SOCKET_TIMEOUT`? That is already at 10 min. I don't know if we need to increase it.